infrastructure/cdn-in-a-box/optional/grafana/trafficcontrol-scenes/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## 1.0.0 (Unreleased)
+
+Initial release.
```

Comment:
I don't think we need this file, do we? Surely anything you'd put in here could just go in the project's CHANGELOG? Which probably wouldn't be much.